### DISCUSSION TITLE
Zipped disk image support?

### DISCUSSION THREAD

**Discussion by charabaruk** [86Box/86Box#4006] [Ideas]
Seems to be fairly common these days for emulators to support zipped ROM or disk images, and all the image managers I know tend to default to zipping or tzipping everything. It would be helpful if 86Box would support attaching zipped disk images and zipped ISOs to VMs so that I don't have to manually unzip everything I want to use into a temp directory every time.

**Comment by lotharsm** [comment]
I'd like to support this request. I keep all my CD dumps (CUE/BIN) in .zip files, and currently, I have to pull them from my archive and extract them to a different place.

With support for zipped CUE/BIN or ISO images, I could attach the files directly from my archive.
